subprojects/core/src/main/groovy/org/gradle/api/artifacts/dsl/RepositoryHandler.java

```diff
@@ -123,6 +123,23 @@
     MavenArtifactRepository jcenter();
 
     /**
+     * Adds a repository which looks in the Gradle's official repository for dependencies. The URL used to access this repository is
+     * {@value org.gradle.api.internal.artifacts.dsl.DefaultRepositoryHandler#GRADLE_REPO_URL}. The name of the repository is
+     * {@value org.gradle.api.internal.artifacts.dsl.DefaultRepositoryHandler#DEFAULT_GRADLE_REPO_NAME}.
+     *
+     * <p>Examples:
+     * <pre autoTested="">
+     * repositories {
+     *     gradleRepo()
```

Comment:
Sounds good. I can make it `pluginRepo` instead of `gradleRepo`.

So it would look like:

```
repositories {
   pluginRepo()
}
```